sites/sample.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+$obj = new clazz();
+
+echo "Its ok";
+
+print_r($_SERVER['nokey']);
+
+
+// daskhdkhsadkhakdhakhsdkahsdkhasdjaldmwqldmlmdlmasldmsalmdalsmdlaksmdlsamdlamsdlmasldmaldmalsmdlasmdlsamdlamsdlamsdlmasldmasldmalsdmalsmdlasmdlasmdlasmdlmasldmaldmad
+
+
+$have_time = true;
+$have_money = false;
+$take_vacation = $have_time and $have_money;
+
+function run() {
+  prepare('action1');                              // Non-Compliant - 'action1' is duplicated 3 times
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "action1" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)